components/general/CustomIcons.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import SvgIcon from 'material-ui/SvgIcon';
+
+const HomeIcon = (props) => (
```

Comment:
Remove this CustomIcons class. I know this icon is being used in two different files but I don't think it is a good idea to start a CustomIcons for now

Reply:
Btw why don't we use a regular material ui icon? See https://material.io/tools/icons/?icon=home&style=baseline